static/app/views/performance/queues/charts/latencyChart.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+
+import {DEFAULT_RELATIVE_PERIODS} from 'sentry/constants';
+import {CHART_PALETTE} from 'sentry/constants/chartPalette';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {CHART_HEIGHT} from 'sentry/views/performance/database/settings';
+import {useQueuesTimeSeriesQuery} from 'sentry/views/performance/queues/queries/useQueuesTimeSeriesQuery';
+import Chart, {ChartType} from 'sentry/views/starfish/components/chart';
+import ChartPanel from 'sentry/views/starfish/components/chartPanel';
+
+interface Props {
+  error?: Error | null;
+}
+
+export function LatencyChart({error}: Props) {
+  const {query} = useLocation();
+  const destination = decodeScalar(query.destination);
+  const pageFilters = usePageFilters();
+  const period = pageFilters.selection.datetime.period;
+  const chartSubtext = (period && DEFAULT_RELATIVE_PERIODS[period]) ?? '';
+  const {data, isLoading} = useQueuesTimeSeriesQuery({destination});
+  return (
+    <ChartPanel title={t('Avg Latency')}>
+      <ChartSubtext>{chartSubtext}</ChartSubtext>
```

Comment:
`ChartPanel` supports a `subtitle` prop, does that help in this case?

Reply:
👍 good call, updated to use subtitle